lib/mincer/engines/sass_engine.js

```diff
@@ -79,5 +85,58 @@ SassEngine.prototype.evaluate = function (context/*, locals*/) {
 };
 
 
+// Returns the argument of the @import() call relative to the asset search paths.
+function importArgumentRelativeToSearchPaths(importer, importArgument, searchPaths) {
+  var importAbsolutePath = path.resolve(path.dirname(importer), importArgument);
+  var importSearchPath = _.find(searchPaths, function(path) {
+    return importAbsolutePath.indexOf(path) === 0;
+  });
+  if (importSearchPath) {
+    return path.relative(importSearchPath, importAbsolutePath);
+  }
+}
+
+
+function isFileNotFound(error) {
+  return error && error.code === 'FileNotFound';
+}
+
+
+SassEngine.prototype.sassImporter = function (context, url, prev) {
+
+  function tryDepend(importPath) {
```

Comment:
I would like this to be moved out of `sassImporter` function (to avoid this function creation upon each request):

``` javascript
function tryDepend(context, importPath) {
  // ...
}

SassEngine.prototype.sassImporter = function (context, url, prev) {
  // ...
  var firstError = tryDepend(context, importPath);
  var secondError;
  if (isFileNotFound(firstError)) {
    secondError = tryDepend(context, underscoredImportPath);
  }
```